tools/pipelines/test-real-service.yml

```diff
@@ -16,7 +16,7 @@ parameters:
 
 - name: poolBuild
   type: object
-  default: Lite
+  default: Main
```

Comment:
@curtisman is this ok? With Lite, building takes 40 minutes as opposed to under 4 with Main.